src/log.rs

```diff
@@ -0,0 +1,105 @@
+// These re-exports of the log crate make it easy to use this crate without having to depend on the
+// log crate directly. See <https://github.com/clap-rs/clap-verbosity-flag/issues/54> for more
+// information.
+pub use log::{Level, LevelFilter};
+
+use crate::{LogLevel, Verbosity, VerbosityFilter};
+
+impl<L: LogLevel> Verbosity<L> {
+    /// Get the log level.
+    ///
+    /// `None` means all output is disabled.
+    pub fn log_level(&self) -> Option<Level> {
+        self.filter().into()
+    }
+
+    /// Get the log level filter.
+    pub fn log_level_filter(&self) -> LevelFilter {
+        self.filter().into()
+    }
+}
+
+impl From<VerbosityFilter> for LevelFilter {
```

Comment:
Hmm, we should have `cfg` on  these impls as well for rustdoc

Reply:
The cfg is at the module level and works correctly for docs-rs